### ISSUE TITLE
Accessibility: ComboBox: Narrator doesn't announce the selected option when using up/down arrows on focused, collapsed, readonly ComboBox

### ISSUE THREAD

**Issue by JllyGrnGiant** [OfficeDev/office-ui-fabric-react#5360]
### Bug Report 
 
- __Package version(s)__: 5.112.0 (and in more recent releases such as 6.21.0)
- __Browser and OS versions__: Edge and Windows 10
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
No
 
Requested priority: (Blocking, High, Normal, Low)
Normal
 
Products/sites affected:
VSTS
 
#### Describe the issue:
1. Open Narrator
2. Open Edge and navigate to a page with a Fabric ComboBox with allowFreeform set to ```false```
3. Tab to the ComboBox
4. Use the up/down arrows to change the selected value of the ComboBox
 
#### Actual behavior:
Narrator doesn't announce the values being selected
 
#### Expected behavior:
Narrator announces the selection changes
 
### If applicable, please provide a codepen
[Repro](https://codepen.io/anon/pen/PayMqp)

**Comment by JasonGore** [comment]
An [ARIA reference example](http://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.0pattern/combobox-autocomplete-list.html) exhibits partially the same behavior in that down arrow doesn't read the highlighted item, but up arrow works. The ARIA specification doesn't exactly have a role like read-only `combobox` as `listbox` role seems more apt in this situation. Either way, it looks like we can at least reach parity with the ARIA example before passing this off as a Narrator issue.

**Comment by JasonGore** [comment]
This appears to have something to do with the autocomplete and selection behavior (or lack thereof) affected by the state of `allowFreeform`. This is also evident when running Narrator in Edge and seeing its selected text and focus behavior non-freeform examples as compared to freeform examples.

**Comment by dzearing** [comment]
@jspurlin Is this a real bug? It's been assigned to @chang47 for a while now, but hasn't been addressed.  Can you look and see if this is a real issue, and what needs to be fixed? This surely impacts everyone using ComboBox.

**Comment by shivasai09** [comment]
@dzearing  is there any update on this issue? M365AdminUx also uses this component, and facing this issue.

**Comment by dzearing** [comment]
I've pinged @jspurlin on this issue.

**Comment by jspurlin** [comment]
@dzearing I'm not able to repro this with Edge+Narrator or Chrome+NVDA, the value gets read when it's focused for me and when the values are switched. The result is inline with how a basic HTML select works: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select

**Comment by shivasai09** [comment]
@jspurlin , here is the  repro url
https://codepen.io/anon/pen/eqgwjd?&editable=true

 

1. please open the above url in edge and switch on the narrator

2. it has two combo boxes, fot the 1st combo box the narrator will not announce when values are changed via up/down arrow, and for 2nd combo box the narrator will announce.


please let me know if the above url replicates this issue

**Comment by jspurlin** [comment]
Again, it reads out just fine for me. I am not able to repro the non-readout behavior. Make sure Narrator is behaving correctly when testing (my require Narrator to be shutdown and restarted)

**Comment by shivasai09** [comment]
hellow @jspurlin  this is also repro in fabric website itself. below is the link for the video
https://drive.google.com/file/d/1cZ4watT8LJSHjBuW2NAnOjZ1aglhMt3o/view?usp=sharing

**Comment by jspurlin** [comment]
@shivasai09 this doesn't align with what I was seeing before. Seems like it's a bug in Narrator. Chrome/NVDA does not repro this problem. I feel like the next steps would be to follow up with the Narrator team

**Comment by shivasai09** [comment]
@jspurlin the narrator is not working only for the comboBox's where it's prop allowFreeform is set to false.
in the above video you can observe that it worked properly for other combobox, because it's allowFreeform is set to true.
if it was a narrator bug then it should behave same for all the combobox, but it din't , i strongly believe that it has to do something with allowFreeForm prop.

**Comment by jspurlin** [comment]
I think it has to do with how Narrator is handling comboboxes differently depending on if allowFreeform is true or false (e.g. if the input on combobox has readonly or not), not how the combobox itself is behaving. This is backed up by the fact that Chrome (or chromium edge) + NVDA work as expected

**Comment by dzearing** [comment]
@aneeshack4 is there someone here we can tag from the Narrator team to confirm?

**Comment by aneeshack4** [comment]
@dzearing contacted them over email yesterday & cc'd you - hoping to get them on Github
